pkgs/games/prismlauncher/wrapper.nix

```
@@ -21,44 +25,74 @@
 , msaClientID ? null
 , gamemodeSupport ? stdenv.isLinux
 , textToSpeechSupport ? stdenv.isLinux
+
+# The flag `withWaylandGLFW` enables runtime-checking of `WAYLAND_DISPLAY`;
+# if the option is enabled, a patched version of GLFW will be added to
+# `LD_LIBRARY_PATH` so that the launcher can use the correct one
+# depending on the desktop environment used.
+, withWaylandGLFW ? false
```

```suggestion
  # The flag `withWaylandGLFW` enables runtime-checking of `WAYLAND_DISPLAY`;
  # if the option is enabled, a patched version of GLFW will be added to
  # `LD_LIBRARY_PATH` so that the launcher can use the correct one
  # depending on the desktop environment used.
, withWaylandGLFW ? false
```

minor formatting nitpick, otherwise lgtm

---

This is the weirdest nitpick you've made so far and I have never seen formatting like that (indented comment lines) in any Nix code before. Since this is Nixpkgs, I'm going to follow the same style of the rest of the files in this repository.

If you're going to review like this, please include some evidence as to why your preferences are the most correct.